modules/ROOT/pages/dataweave-formats-avro.adoc

```diff
@@ -99,36 +103,38 @@ output application/avro schemaUrl="classpath://schema.json"
 ----
 
 
-// CONFIG PROPS ///////////////////////////////////////////////////////
-
 [[properties]]
 == Configuration Properties
 
-DataWeave supports the following configuration properties for Avro.
+DataWeave supports the following configuration properties for the Avro format.
```

Comment:
TODO: everywhere change "for the <format_name> format" -> "for this format" due to issues with names of some formats, like "Dw"